src/backend/TrafficCourts/TrafficCourts.Staff.Service.Test/Controllers/ComsControllerTest.cs

```diff
@@ -28,8 +28,8 @@ public async void TestUploadDocument200Result()
         comsService
             .Setup(_ => _.SaveFileAsync(It.IsAny<IFormFile>(), It.IsAny<Dictionary<string, string>>(), It.IsAny<CancellationToken>()))
             .ReturnsAsync(guid);
-        var mockLogger = new Mock<ILogger<ComsController>>();
-        ComsController comsController = new(comsService.Object, mockLogger.Object);
+        var mockLogger = new Mock<ILogger<DocumentController>>();
+        DocumentController comsController = new(comsService.Object, mockLogger.Object);
 
         // Act
         IActionResult? result = await comsController.UploadDocumentAsync(mockFileUploadRequest.Object, CancellationToken.None);
```

Comment:
It would be nice, however, if there are more pressing issues to address. This was a new method I added so I could avoid updating all your unit tests.

Reply:
The `Could not load file or assembly` issue has been addressed by explicitly adding a reference to the target projects. The issue is due to transitive dependency pulling in version 6.0.12.0 instead of 6.0.13.0. We wouldn't have this issue if we were on .NET 7.